packages/cli/src/util/link/ensure-link.ts

```diff
@@ -34,7 +34,7 @@ export async function ensureLink(
 ): Promise<LinkResult | number> {
   let link = await getLinkedProject(client, cwd);
 
-  if (link.status === 'not_linked') {
+  if (opts.forceDelete || link.status === 'not_linked') {
```

Comment:
It's possible that `link.status` will equal `"error"`, so you need to explicitly check the link status is `"linked"` in addition to `opts.forceDelete`:

```suggestion
  if ((link.status === 'linked' && opts.forceDelete) || link.status === 'not_linked') {
```